src/modules/create/game-editor/editors/CodeEditor.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useRef } from "react";
+import Editor, { Monaco } from "@monaco-editor/react";
+import { editor } from "monaco-editor";
+import CodeTabTheme from "./CodeTabTheme";
+import { MonacoBinding } from "y-monaco";
+import { EditorProps } from "./EditorType";
+
+const CodeEditor: React.FC<EditorProps> = ({ ydoc, provider }) => {
+  const monacoBindingRef = useRef<MonacoBinding | null>(null);
+
+  const handleMount = (editor: editor.IStandaloneCodeEditor): (() => void) | void => {
+    const editorModel = editor.getModel();
+    if (!editorModel) {
+      console.error("Monaco editor model is not available.");
+      return;
+    }
+
+    monacoBindingRef.current = new MonacoBinding(
+      ydoc.getText("monaco"),
+      editorModel,
+      new Set([editor]),
+      provider.awareness
+    );
+
+    return () => {
+      editor.dispose();
+      monacoBindingRef.current?.destroy?.();
+    };
+  };
+
+  const handleBeforeMount = (monaco: Monaco): void => {
+    monaco.editor.defineTheme(CodeTabTheme.MONACO_THEME_NAME, CodeTabTheme.MONACO_THEME);
+  };
+
+  return (
+    <>
+      <style>{`
+        .yRemoteSelection { 
```

Comment:
I think you should proceed like so: https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/
This would avoid us to rely on a global `style` block which will be cumbersome soon or later.